test/specs/test-error-list.el

```diff
@@ -137,9 +137,8 @@
       (it "has the error ID in the 4th cell"
         (expect (aref cells 3) :to-equal
                 (list "W1"
-                      'type 'flycheck-error-list
-                      'face 'flycheck-error-list-id
-                      'help-echo "W1")))
```

Comment:
Hmm. We just added that help-echo feature in https://github.com/flycheck/flycheck/issues/1101, so I'm not sure about removing it :)